### PR TITLE
create variant for magicbit board

### PR DESCRIPTION
**Pull request by Magicblocks**

### CODE REVIEW

**Comment by Magicblocks** [comment]
Added magicbit board to variant folder and changed board.txt file
image

File: boards.txt [hunk 1]
```diff
@@ -230,7 +230,41 @@ esp32wrover.menu.DebugLevel.verbose=Verbose
 esp32wrover.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
-
+magicbit.name=MagicBit
```
Comment (me-no-dev) [inline boards.txt]: you need to move this below the PICO board. Top 3 boards are the official chips and modules from Espressif :)

**Comment by Magicblocks** [comment]
@me-no-dev  changed the order

**Comment by me-no-dev** [comment]
@Magicblocks done :)

**Pull request merged**